src/procedures/SetDividendsWallet.ts

```diff
@@ -47,6 +58,9 @@ export class SetDividendsWallet extends Procedure<SetDividendsWalletProcedureArg
         );
         break;
       }
+      default: {
```

Comment:
Why was this added?

Reply:
TypeScript recommendation

Reply:
What do you mean by that? Which tool recommended it exactly? The compiler says nothing